resources/views/admin/user/create.blade.php

```diff
@@ -0,0 +1,32 @@
+@extends('admin.master')
+@section('content')
+<h1>Create user</h1> 
```

Comment:
```suggestion
<h1>trans('create_user')</h1> 
```
sử dụng i18n

với `create_user => 'Create user'`